src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
 }
+
+:root {
+  --blue_link_color: #00acdc;
+}
+
+header {
+  padding-right: 50px;
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+li {
+  list-style-type: none;
+  margin-left: 20px;
+}
+
+li:first-child {
+  margin-left: 0;
+}
+
+.moyoIcon {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.navigationList {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav_link {
+  display: block;
+  font-size: 12px;
+  line-height: 60px;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav_link:hover {
+  color: var(--blue_link_color);
+}
+
+[data-qa]:hover {
```

Comment:
No need to use attributes as selector, use classes selector instead

Reply:
There is an instruction in "Readme" 
and it requires to use "data-qa"
should I just ignore it?